TabsPls_Qt/LightSpeedFileExplorer/FileListTableViewWithFilter.cpp

```diff
@@ -54,6 +54,9 @@ FileListTableViewWithFilter::FileListTableViewWithFilter(std::weak_ptr<CurrentDi
     m_fileListTableView = fileListTableView;
     m_fileListTableView->setModel(&viewModel);
 
+    m_fileListTableView->resizeColumnsToContents();
```

Comment:
Better put this higher up, this is the filter class, it should not set up anything else